src/Resources/views/CRUD/Association/edit_many_to_one.html.twig

```diff
@@ -24,11 +24,11 @@ file that was distributed with this source code.
                         'code':     sonata_admin.field_description.associationadmin.code,
                         'objectId': sonata_admin.field_description.associationadmin.id(sonata_admin.value),
                         'uniqid':   sonata_admin.field_description.associationadmin.uniqid,
-                        'linkParameters': sonata_admin.field_description.options.link_parameters
+                        'linkParameters': sonata_admin.field_description.option('link_parameters')
                     })) }}
-                {% elseif sonata_admin.field_description.options.placeholder is defined and sonata_admin.field_description.options.placeholder %}
+                {% elseif sonata_admin.field_description.option('placeholder') %}
```

Comment:
As a general rule, should we change from:
```twig
variable.field_description.options.some_option is defined and variable.field_description.options.some_option 
```
to
```twig
variable.field_description.option('some_option') is not null
```
to be explicit?

Reply:
I don't think so, because if `variable.field_description.option('some_option') is not null` will evaluate to `true` when the "some_option" option is set with the value `false`, and in this example, the condition is trying to check if the option is holding a truly value.